index.js

```diff
@@ -61,7 +61,7 @@ function formatWithShebang(text, opts) {
   const index = text.indexOf("\n");
   const shebang = text.slice(0, index + 1);
   const programText = text.slice(index + 1);
-  return shebang + format(programText, opts);
+  return shebang + "\n" + format(programText, opts);
```

Comment:
@vjeux is right. We should follow the blank line rule like we do elsewhere. Here you can check if `index + 1` is a newline, and if so, add a newline (don't forget, for windows support the check must be `char === "\n" || char === "\r"`.